ludwig/api.py

```diff
@@ -60,6 +60,10 @@
 from ludwig.utils.print_utils import logging_level_registry
 
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
You don't need to set the level on each individual logger as log message propagate to their parent by default. So a logger `ludwig.api` will sends received messages to the logger `ludwig` that is configured on the right right log level.

Reply:
In this case, we wanted to set specific levels for different modules.